cfn-resources/search-deployment/test/README.md

```diff
@@ -0,0 +1,44 @@
+# MongoDB::Atlas::SearchDeployment
+
+## Impact 
+The following components use this resource and are potentially impacted by any changes. They should also be validated to ensure the changes do not cause a regression.
+ - SearchDeployment L1 CDK constructor
+
+
+## Prerequisites 
+### Resources needed to run the manual QA
+All resources are created as part of `cfn-testing-helper.sh`:
+
+- Atlas Project
+- Cluster
+
+## Manual QA
+Please follow the steps in [TESTING.md](../../../TESTING.md).
+
+
+### Success criteria when testing the resource
+1. After successful creation of the stack using a template from examples section, by seeing the details of the Cluster configuration (https://cloud-dev.mongodb.com/v2/<project-id>#/clusters/detail/Cluster0) the isolated search nodes should appear together with the existing dedicated cluster nodes.
```

Comment:
I left this comment in other PRs as well, not sure if I am missing something. Why do we specify `cloud-dev`?

Reply:
No strong reason, when testing the resource it is likely that you will be using dev over prod.

Reply:
i agree that this can be confusing for users to have in a README file

Reply:
adjusted to avoid any confusion, would keep in mind that ./test/README.md files are internal for us to keep record of how each resource can be tested.